XMonad/Prompt/OrgMode.hs

```diff
@@ -382,7 +387,11 @@ toOrgFmt tod day =
   mconcat ["<", isoDay, " ", take 3 $ show (dayOfWeek day), time, ">"]
  where
   time   :: String = maybe "" ((' ' :) . show) tod
+#if MIN_VERSION_time(1, 9, 0)
+  isoDay :: String = iso8601Show day
+#else
   isoDay :: String = formatTime defaultTimeLocale (iso8601DateFormat Nothing) day
+#endif
```

Comment:
I guess this is one vote for dropping 8.6 as well in the near-ish future, but not sure if it's enough (the module has to use `CPP` anyways for the exports when testing so...)